sbt-plugin/src/main/scala/org/scalajs/sbtplugin/CacheBox.scala

```diff
@@ -17,18 +17,20 @@ package org.scalajs.sbtplugin
  *  A CacheBox is needed, once the cached result needs to depend on a task,
  *  since then it cannot simply be made a setting anymore.
  */
-private[sbtplugin] final class CacheBox[T] {
+final class CacheBox[T] {
```

Comment:
Can we make only the type publicly accessible (not it's methods, constructor, etc.)?

Reply:
Not if we keep the contract that custom `scalaJSLinkerImpl`s must call `ensure` on `scalaJSLinkerImplCacheBox.value`...